bats/README.md

```diff
@@ -151,3 +151,11 @@ After finishing to develop a BATS test suite, you can locally verify the syntax
      ```sh
      shfmt -w ./bats/tests/containers/factory-reset.bats
      ```
+## Other tips
+
+Use `$BATS_TMP_DIR` for temporary files. On Windows this needs to be converted to a wsl-compatible path
```

Comment:
There is no `$BATS_TMP_DIR` variable.

```suggestion
Use `$BATS_FILE_TMPDIR` for temporary files. On Windows this needs to be converted to a wsl-compatible path
```

I would rather point to https://bats-core.readthedocs.io/en/stable/writing-tests.html#special-variables than to recommend a specific variable here. In general I don't think we should try to repeat the BATS docs, but rather point to the upstream docs instead.